examples/kubernetes/static_provisioning/s3_express_specify_az.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: s3-pv
+spec:
+  capacity:
+    storage: 1200Gi # ignored, required
+  accessModes:
+    - ReadWriteMany # supported options: ReadWriteMany / ReadOnlyMany
+  mountOptions:
+    - allow-delete
+    - region us-west-2
+  csi:
+    driver: s3.csi.aws.com # required
+    volumeHandle: s3-csi-driver-volume
+    volumeAttributes:
+      bucketName: s3-csi-driver--usw2-az1--x-s3
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: s3-claim
+spec:
+  accessModes:
+    - ReadWriteMany # supported options: ReadWriteMany / ReadOnlyMany
+  storageClassName: "" # required for static provisioning
+  resources:
+    requests:
+      storage: 1200Gi # ignored, required
+  volumeName: s3-pv
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: s3-app
+spec:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+          - matchExpressions:
+            - key: topology.kubernetes.io/zone
+              operator: In
+              values:
+              - us-west-2a # specify az where bucket lives
```

Comment:
do you know if there's a selector that supports AZ IDs like `usw2-az2`? otherwise folks will need to look up the mapping between the AZ ID of their directory bucket and the AZ name in their account (https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html), which is maybe worth a comment

Reply:
nope, that's not supported here so I added a reference to this [link](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) since it indicates all the codes that can be used here -- we'll make sure to include support for AZ ID when we work on [this issue](https://github.com/awslabs/mountpoint-s3-csi-driver/issues/93)